docs/docusaurus/docs/reference/learn/data_quality_use_cases/schema.md

```diff
@@ -1,8 +1,246 @@
+:::
+
+### Expect Table Columns to Match Set
+
+Checks that the dataset contains specific columns, without regard to order. This Expectation offers
+flexibility where column presence is more critical than their sequence. ([Link
+to Expectation Gallery](https://greatexpectations.io/expectations/expect_table_columns_to_match_set))
+
+```python
+gxe.ExpectTableColumnsToMatchSet(column_set=[
+  "sender_account_number", "recipient_account_number", "transfer_amount", "transfer_date"
+], exact_match=False)
+```
+
+:::info[Use Case]
+Useful for datasets that might undergo reordering during preprocessing; key for data
+warehousing operations where column integrity is crucial, but order might vary.
+:::
+
+:::tip[GX Tip]
+Opt for `ExpectTableColumnsToMatchSet` when integrating datasets from
+various sources where column order might differ, but consistency in available data is required.
+:::
+
+### Expect Table Column Count to be Between
+
+Validates that the number of columns falls within a specific range, offering flexibility for
+datasets that can expand or contract within a known boundary. ([Link
+to Expectation Gallery](https://greatexpectations.io/expectations/expect_table_column_count_to_be_between))
+
+```python
+gxe.ExpectTableColumnCountToBeBetween(min_value=6, max_value=8)
+```
+
+:::info[Use Case]
+Beneficial for evolving datasets where additional columns could be added over time,
+but the general structure remains bounded within a predictable range.
+:::
+
+:::tip[GX Tip]
+Regularly review the allowed range as your dataset evolves, ensuring it aligns
+with business requirements and anticipates potential future expansion.
+:::
+
+## Examples and scenarios
+
+Successful schema validation can be accomplished using either GX Cloud or the GX Python library. The examples and scenarios in this section are presented using the GX Python library, but can be implemented as Cloud workflows to achieve the same result.
+
+![Validate schema Expectations in GX Cloud](./images/gx_cloud_schema_expectations_validate.gif)
+
+### Comparative analysis: Ensuring schema consistency in financial transfers
+
+**Context**: In financial transfers, adhering to a fixed schema is paramount for regulatory compliance and operational accuracy. Ensuring that all necessary columns are present and correctly typed can prevent significant operational disruptions.
+
+**Goal**: Validate two datasets to ensure the presence of specific columns and correct column count.
+
+```python title="Python" name="docs/docusaurus/docs/reference/learn/data_quality_use_cases/schema.py full sample code"
+```
+
+**Insight**: Dataset 2 fails to validate due to the absence of `recipient_fullname` in one of the rows and the correct column count, highlighting how missing critical columns can disrupt financial processing or lead to compliance issues.
+
+### Different Expectation suites: Strict vs. relaxed type checking
```

Comment:
I think the code recipe in this section could use some improvement, here are my suggestions:
* Validate a single dataset using two different Expectation Suites to highlight the difference in results, instead of validating two datasets with two different Expectation Suites.
* The two separate examples are mostly duplicated code, with the exception of the Expectation Suites, which makes this section difficult to parse as a reader. Can you please consolidate the code into a single workflow (script) that defines the Data Source (using one dataset) and two Expectation Suites that are then run against the same Data Source? I think naming the Suites `strict_suite` and `relaxed_suite` (or something similar - feel free to use what naming you think best) will make it clearer to the user what we're trying to communicate.
* I think it would be beneficial to also include `ExpectColumnValuesToBeOfType` and `ExpectColumnValuesToBeInTypeList` in the example Suites to illustrate strict vs. relaxed validation, respectively.

Reply:
@ixf-tdg4gx As part of refactoring the organization of the schema code under test so that we contain all the schema-related scripts in a subdirectory, I consolidated the relaxed and strict validation examples into a single script: `docs/docusaurus/docs/reference/learn/data_quality_use_cases/schema_resources/schema_strict_and_relaxed.py`
If you could tackle the third bullet (adding type checking to the strict/relaxed suites in addition to column set), that would be great. Thanks!